app/vendors/vendor-dashboard/profile/VendorProfileForm.tsx

```
@@ -0,0 +1,338 @@
+    message: "City must be at least 2 characters.",
+  }),
+  state: z.string().min(2, {
+    message: "State must be at least 2 characters.",
+  }),
+  serviceCategories: z.array(z.number()),
+});
+
+type FormValues = z.infer<typeof formSchema>;
+
+interface VendorProfileFormProps {
+  vendor: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)